plugins/meta/flannel/flannel_linux.go

```diff
@@ -78,10 +78,18 @@ func doCmdDel(args *skel.CmdArgs, n *NetConf) error {
 		return err
 	}
 
+	defer func() {
+		// remove net conf after DEL succeed
+		if err == nil {
```

Comment:
To make the return of `invoke.DelegateDel` can be judged here, @jellonek

Reply:
You could avoid this by having `(err error)` as the return, i.e. a named return variable, and changing `err :=` to `err =`.  Both are pretty subtle points of Go behaviour.

Reply:
thanks, will update this